adapters/vue/.eslintrc

```diff
@@ -0,0 +1,35 @@
+{
+  "parserOptions": {
```

Comment:
Do we need to define "parserOptions" again given its in the base (ie extends)

Reply:
Yeah I was hoping that would be the case but based off the docs, since we are also extending plugin:vue/recommended :) 

"If you already use other parser (e.g. "parser": "babel-eslint"), please move it into parserOptions, so it doesn't collide with the vue-eslint-parser used by this plugin's configuration" https://vuejs.github.io/eslint-plugin-vue/user-guide/#faq